app/services/support_interface/persona_export.rb

```diff
@@ -37,7 +37,15 @@ def data_for_export(*)
 
     def application_choices
       ApplicationChoice
-      .includes(%i[application_form site provider course candidate])
+      .includes(
+        [
+          :provider,
+          :site,
+          { application_form: [{ application_choices: [:site] }, :application_qualifications] },
+          { course: [:accredited_provider] },
+          { course_option: [:site, { course: [:provider] }] },
```

Comment:
out of interest, are we also interested in `current_course_option`?

Reply:
good catch, seems like this export is only looking at course (through course option) for some reason. We could change it to current course and use that instead?